Makefile

```diff
@@ -21,5 +22,10 @@ publish:
 	@echo ""
 	@echo "Next steps:"
 	@echo " - add new version to CHANGELOG.md"
-	@echo " - push changes to repository by : \"git commit -am 'prepare to next development iteration' && git push\""
+	@echo " - push changes to repository by : \"git commit -am 'chore(release): prepare to next development iteration [skip CI]' && git push\""
 	@echo ""
+
+nightly:
+	yarn install --frozen-lockfile
+	yarn run build
```

Comment:
we might signalize a nightly build
```sh
sed  -i '' -e "s/CLIENT_LIB_VERSION = '\([0-9.]*\).*'/CLIENT_LIB_VERSION = '\1 nightly'/" packages/core/src/impl/version.ts
```

Reply:
I prepared script that update `CLIENT_LIB_VERSION` to same version as is used by `lerna --canary`.

`version.ts` looks like:

```typescript
export const CLIENT_LIB_VERSION = '1.2.0.nightly'
```

builded packages:

```bash
Found 2 packages to publish:
 - @influxdata/influxdb-client-apis => 1.2.0-nightly.9+dd9bb60
 - @influxdata/influxdb-client => 1.2.0-nightly.9+dd9bb60
```